kolibri/plugins/learn/assets/src/views/AlsoInThis.vue

```diff
@@ -9,7 +9,7 @@
       <KRouterLink
         v-for="content in contentNodes"
         :key="content.id"
-        :to="genContentLink(content.id, null, content.is_leaf, null, context)"
+        :to="genContentLink(content.id, content.is_leaf, true)"
```

Comment:
I guess my thought around passing an object instead would make this a bit gross and annoying to format -- especially if using a longer property name for the third param

Reply:
I guess we could make a computed property for the object we need here if the function took an object instead